pkg/getty/usr/bin/rungetty.sh

```diff
@@ -41,6 +41,14 @@ start_getty() {
 	infinite_loop setsid.getty -w /sbin/agetty $loginargs $line $speed $tty $term &
 }
 
+
+# check if we are namespaced, and, if so, indicate in the PS1
+if [ -z "$INIGETTY" ]; then
```

Comment:
Whatever, I don't care. Fixing, pushing. Of course that means CI will be another week until it runs through.